ingest-app/src/cmr/ingest/migrations/011_create_granule_bulk_update_tables.clj

```diff
@@ -0,0 +1,48 @@
+(ns cmr.ingest.migrations.011-create-granule-bulk-update-tables
+  (:require
+   [config.ingest-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up to version 11."
+  []
+  (println "cmr.ingest.migrations.011-create-granule-bulk-update-tables up...")
+  (h/sql
+   (str "CREATE TABLE CMR_INGEST.granule_bulk_update_tasks ("
+        "TASK_ID VARCHAR2(255) NOT NULL, "
+        "PROVIDER_ID  VARCHAR2(10) NOT NULL, "
+        "NAME VARCHAR2(255) NOT NULL, "
```

Comment:
Should we default this to the `TASK_ID`, Or do we want to have the code handle this?

Reply:
the code.